charts/pega/values.yaml

```diff
@@ -213,3 +214,9 @@ installer:
     upgradeType: "in-place"
     # Specify target rules schema for migration and upgrade
     targetRulesSchema: ""
+
+# Pega Constellation UI settings.
+# Note : This node is for constellation service whose purpose is to serve static content to applications.
+# This is a feature currently under development and is to be used only with Pega versions 8.5 and above.
+constellation:
```

Comment:
Remove and merge as a separate pull request when we are ready to ship the service.  The remainder of this PR can be merged in a dormant state.

Ensure `constellation.enabled` defaults to `false`.